aws-predictions-tensorflow/src/main/java/com/amplifyframework/predictions/tensorflow/service/TensorFlowTextClassificationService.java

```diff
@@ -62,13 +64,23 @@
      * @param context the Android context
      */
     TensorFlowTextClassificationService(@NonNull Context context) {
```

Comment:
One of the reason I user factories all the time is that they allow you to write code above the equivalent of `this(...)`. e.g.,

```
static TensorFlowTextClassificationService fromContext(@NonNull Context context) {
    Objects.requireNonNull(context);
    TextClassificationModel model = new TextClassificationModel(context);
    TextClassificationDictionary dictionary = new TextClassificationDictionary(context);
    TextClassifcationLabels labels = new TextClassificationLabels(context);
    return new TensorFlowTextClassificationService(model, dictionary, labels);
}
```

Overall, you just don't have to fight the syntax as hard, with this approach ^^ as compared the constructor-way.